docs/rest/src/_fragments/activities/GrantUserAccessResponse.json

```diff
@@ -0,0 +1,15 @@
+{
+  "GrantUserAccessResponses": [
+    {
+      "Id": "123123123",
+      "Provider": "wopi-contoso",
+      "Status": 1,
+      "Message": "User has insufficient permissions to grant other users access"
```

Comment:
In this hypothetical scenario does the other one succeed because the first was an external user or something?